src/WSSESoap.php

```diff
@@ -281,30 +403,32 @@ public function signSoapDoc($objKey, $options = null)
             $algorithm = $options['algorithm'];
         }
 
-        $arOptions = array('prefix' => self::WSUPFX, 'prefix_ns' => self::WSUNS);
+        $arOptions = ['prefix' => self::WSUPFX, 'prefix_ns' => self::WSUNS];
         $objDSig->addReferenceList($arNodes, $algorithm, null, $arOptions);
 
         $objDSig->sign($objKey);
 
         $insertTop = true;
         if (is_array($options) && isset($options['insertBefore'])) {
-            $insertTop = (bool) $options['insertBefore'];
+            $insertTop = (bool)$options['insertBefore'];
         }
         $objDSig->appendSignature($this->secNode, $insertTop);
 
         /* New suff */
         if (is_array($options)) {
             if (!empty($options['KeyInfo'])) {
                 if (!empty($options['KeyInfo']['X509SubjectKeyIdentifier'])) {
-                    $sigNode = $this->secNode->firstChild->nextSibling;
+                    $sigNode = $this->secNode->firstChild;
```

Comment:
Why needed?

Reply:
To be honest I do not remember as I made those changes over a year ago, I will have to do some digging and get back to you. Thanks for your time.